installation/configuration.md

```diff
@@ -1,112 +1,133 @@
+The `LEAN_S3_FOLDER_NAME` is optional but can be helpful if you store other objects in the same S3 bucket. After switching to S3, files previously uploaded to the local server will remain there but will no longer be accessible through Leantime.
 
 ## SMTP Configuration
-Leantime supports sending email via the standard `mail()` function or via smtp. To configure SMTP fill in the SMTP section with your host, username, password and the security protocol used.
+
+For outbound emails generally two methods can be used: the standard PHP `mail()` function and SMTP. While the `mail()` function is simpler, SMTP often provides better deliverability and security. The following guide shows process of configuring SMTP.
+
+Navigate to the `config/.env` file and update the SMTP configuration section:
 
 ```php
-public $useSMTP = true; //Use SMTP? If set to false, the default php mail() function will be used
-public $smtpHosts = ""; //SMTP host
-public $smtpUsername =""; //SMTP username
-public $smtpPassword = ""; //SMTP password
-public $smtpSecure =""; //SMTP Security protocol (usually one of: TLS, SSL, STARTTLS)
-public $smtpPort = ""; //Port (usually one of 25, 465, 587, 2526)
+## Email
+LEAN_EMAIL_RETURN=''              # Valid return email address
+LEAN_EMAIL_USE_SMTP=true          # true -> SMTP, false -> php mail()
+LEAN_EMAIL_SMTP_HOSTS=''          # SMTP host
+LEAN_EMAIL_SMTP_AUTH=false        # true -> SMTP authentication required; email and password needed. false -> authentication not required
+LEAN_EMAIL_SMTP_USERNAME=''       # SMTP username
+LEAN_EMAIL_SMTP_PASSWORD=''       # SMTP password
+LEAN_EMAIL_SMTP_AUTO_TLS=true     # SMTP Enable TLS encryption automatically if a server supports it
+LEAN_EMAIL_SMTP_SECURE=''         # SMTP Security protocol (usually one of: TLS, SSL, STARTTLS)
+LEAN_EMAIL_SMTP_SSLNOVERIFY=true  # SMTP Allow insecure SSL: Don't verify certificate, accept self-signed, etc.
+LEAN_EMAIL_SMTP_PORT=''           # Port (usually one of 25, 465, 587, 2526)
 ```
 
-## Ldap Configuration
-As of v2.1.9 Leantime supports a basic ldap integration. A few notes:
-* When a user first logs in a new Leantime user is being created. The default role can be defined in the config file
-* You can map roles from your ldap directory to roles in Leantime using the `$ldapLtGroupAssignments`
-* To get started, set `$useLdap` to true.
-* Instead of php arrays we decided to use json as the value format to allow you to configure ldap via environment variables. Please make sure your json is valid.
+Common issues that users encounter such as if port is closed, check firewall settings to ensure the given port is open. Also, depending on the host, some may require you to use their own SMTP servers.
+
+## LDAP Configuration
+
+Leantime supports basic LDAP integration via configuration of the environament variables. Follow the porcess below to setup LDAP:
+
+Navigate to the `config/.env` file and update the following settings:
```

Comment:
@ctanveer @marcelfolaron 

Probably worth adding a end line here that says,

"LDAP is not supported in Leantime with versions earlier than v2.1.9." 

Thoughts?

Reply:
Makes sense. @ctanveer can you add a "since" tag below the headline?